elm/spec/dependabot/elm/file_fetcher_spec.rb

```diff
@@ -6,6 +6,25 @@
 require_common_spec "file_fetchers/shared_examples_for_file_fetchers"
 
 RSpec.describe Dependabot::Elm::FileFetcher do
+  before do
+    stub_request(:get, url + "?ref=sha")
+      .with(headers: { "Authorization" => "token token" })
+      .to_return(
+        status: 200,
+        body: fixture("github", "contents_elm_with_elm_package.json"),
+        headers: json_header
+      )
+    stub_request(:get, url + "elm-package.json?ref=sha")
+      .with(headers: { "Authorization" => "token token" })
+      .to_return(
+        status: 200,
+        body: fixture("github", "contents_elm_package.json"),
+        headers: json_header
+      )
+  end
+
+  before { allow(file_fetcher_instance).to receive(:commit).and_return("sha") }
```

Comment:
We could combine this setup into the preceding `before` block. (Bonus: helps us start paying down [RSpec/ScatteredSetup](https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecscatteredsetup))